docs/guides/admin/docs/upgrade.md

```diff
@@ -48,6 +48,20 @@ Please make sure to compare your configuration against the current configuration
   [this document](https://github.com/elan-ev/opencast-studio/blob/2020-09-14/CONFIGURATION.md).
 
 
+### Wowza streaming configuration changes
+
+In [pull request #1179](https://github.com/opencast/opencast/pull/1179), the configuration of Wowza servers was made
+tenant-specific. Before, you had to specify `etc/org.opencastproject.streaming.url` in `custom.properties`. Starting
+with Opencast 9, that value has to specified as `org.opencastproject.mh_default_org.wowza.url` in
```

Comment:
Also I'd rather see \<tenant\> in the key instead of "hardcoding" the default tenant. x)